README.md

```diff
@@ -59,6 +61,31 @@ Install specific files in Dom0:
 sudo qubesctl state.apply dotfiles.copy-dom0,dotfiles.copy-sh,dotfiles.copy-vim,dotfiles.copy-x11
 ```
 
+#### Pillar
+
+With Salt, the execution of each state can be opt out via the dotfiles pillar.
+You will need a top and a sls file in your pillar_roots (default to /srv/pillar,
+/srv/pillar/base, can be configured in /etc/salt/minion.d).
```

Comment:
It is is missing empty lines between code blocks, markdown-lint would have failed but I forgot to setup CI for my dotfiles.